framework/web/Request.php

```diff
@@ -547,9 +547,6 @@ public function getQueryParam($name, $defaultValue = null)
      * > If you don't have access to the server configuration, you can setup [[\yii\filters\HostControl]] filter at
      * > application level in order to protect against such kind of attack.
      *
-     * @property string|null schema and hostname part (with port number if needed) of the request URL
-     * (e.g. `http://www.yiiframework.com`), null if can't be obtained from `$_SERVER` and wasn't set.
-     * See [[getHostInfo()]] for security related notes on this property.
```

Comment:
why did you remove this? It sensures that the security hint is also contained in the property, not only getter/setter.